src/index.njk

```diff
@@ -70,20 +70,23 @@
 
 </head>
 <body>
-    <header class="o-header">
-        <a href="/" class="o-header__logo">
-            <img src="images/a-logo.svg" alt="Officieel Logo Antwerpen">
-        </a>
-    </header>
+    <div role="banner">
+        <header class="o-header" aria-label="Header">
+            <a href="#main-content" class="a-button-negative o-header__button o-header__button-skip">Skip to main content</a>
+            <a href="/" class="o-header__logo" aria-label="Antwerp branding styleguide">
+                <img src="images/a-logo.svg" alt="Officieel Logo Antwerpen">
+            </a>
+        </header>
 
-    <main>
         <div class="u-wrapper d-h1 u-text-xlight u-bg-primary">
             <div class="u-container">
                 <h1>Antwerp branding styleguide <small class="u-bg-secondary u-text-xlight">v{{ VERSION_INFO.version }}</small></h1>
                 <p class="u-margin-top">A collection of building blocks for your next Antwerp-based application.</p>
             </div>
         </div>
+    </div>
 
+    <div role="main" id="main-content">
```

Comment:
waarom niet gewoon de <main> tag?

Reply:
Ik vermoed dat je `<main>` bedoelde? Omdat dit element niet (correct) ondersteund wordt door IE11.

Reply:
als je main { display: block; } in uw base toevoegt is IE9+ in orde. Is idd afwegen dan natuurlijk...